packages/web/config/ibc-assets.ts

```diff
@@ -44,7 +44,9 @@ export const IBCAssetInfos: (IBCAsset & {
             AxelarSourceChainConfigs.usdc.fantom,
           ],
         },
-        fiatRamps: [{ rampKey: "kado" as const, assetKey: "USDC" }],
+        fiatRamps: [
+          { rampKey: "layerswapcoinbase" as const, assetKey: "USDC" },
```

Comment:
Removing kado?

Reply:
Yeah.  It's not really a way of depositing USDC.  It's an onramp for USDC, and should instead be moved to a fiat onramp selector along with transak

Reply:
Oh ok. We we're considering a move from bank account > osmosis as depositing, along with bridges/sends from CEXs